src/components/App/App.js

```diff
@@ -1,5 +1,6 @@
 import React, { PropTypes, Component } from 'react';
 import Header from '../common/Header';
+import 'react-select/dist/react-select.css';
```

Comment:
You already imported this line in the `index.js` (https://github.com/juanlacruz/TargetMVD-React/pull/6/files#diff-1fdf421c05c1140f6d71444ea2b27638R13)

It's not necessary to import it again